api_food/foodapi.py

```diff
@@ -0,0 +1,43 @@
+#access the USDA food database, for Take a Byte
+#
+
+import requests #pip3 install requests
+import json 
+
+type = '/v1/foods/search'
+key = '8Wd9vaf4eJdizRTosIiIDPce3jOOloEfQc9aiDM1'
+#url: https://api.nal.usda.gov/fdc/v1/foods/search/?api_key=
+
+params = {
+    'key' : key,
+    'query' : 'cheese',
+    'pageSize' : 5,
+    'pageNumber' : 1,
+    'dataType' : 'Branded',
+    'numberOfResultsPerPage' : 5,
+    
+}
+
+fetch = requests.get(f"https://api.nal.usda.gov/fdc{type}/?api_key={params['key']}&query={params['query']}&dataType={params['dataType']}&pageSize={params['pageSize']}&pageNumber={params['pageNumber']}&numberOfResultsPerPage={params['numberOfResultsPerPage']}")
```

Comment:
Fix: Missing timeout argument for method 'requests.get' can cause your program to hang indefinitelyPylint[W3101:missing-timeout](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/missing-timeout.html)